README.md

```diff
@@ -33,7 +33,7 @@ code for these systems is typically 50 lines or less.
 ## Using DeepQA
 
 To train or evaluate a model using DeepQA, the recommended entry point is to use the
-[`run_model.py`](./scripts/run_model.py) script.  That script takes one argument, which is a
+[`run.py`](./scripts/run.py) script.  That script takes one argument, which is a
```

Comment:
What did you decide on `run.py` vs. `run_model.py`?  If you want it to be `run.py`, there are lots of places below that should be switched.  I think this is the only place that currently has `run.py`.